### PR TITLE
Removed is_docker stuff across roles

### PR DESCRIPTION
**Pull request by adi-mohan**
removed is_docker stuff across roles. Also fixed related tests. #217 

##### SUMMARY

#217 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
 check virtualization type using ansible_facts

### CODE REVIEW

**Comment by rhysmeister** [comment]
@adi-mohan @meysam81

These two PRs have almost the same content. Do you two know each other?

https://github.com/ansible-collections/community.mongodb/pull/218

**Comment by meysam81** [comment]
> @adi-mohan @meysam81
> 
> These two PRs have almost the same content. Do you two know each other?
> 
> #218

No, we don't. But the PR open time proves no explanation. :smiley_cat:

**Comment by rhysmeister** [comment]
@adi-mohan @meysam81

It's not possible to merge both. You may as well both work over on the other issue. I think you can add someone as a 
collaborater...

https://github.com/ansible-collections/community.mongodb/pull/218

**Comment by adi-mohan** [comment]
Hi @rhysmeister @cognifloyd ! I did not want to spam this PR with patch-sets but galaxy is getting timed out as Jacob mentioned here https://github.com/ansible/galaxy/issues/2302#issuecomment-694427651 
Also, can you help me out on why sanity is failing even when number of blank lines are 2 in that file. PEP8 coding guidelines didn't help me much. Thought of asking you guys.

**Comment by rhysmeister** [comment]
Sanity test error...

`ERROR: roles/mongodb_linux/molecule/default/tests/test_default.py:54:5: E303: too many blank lines (2)`

You have 2 lines within this funciton. pep8 likes 1 only within functions.

For the purposes of this PR you can ignore the Galaxy timeouts. Since you're only modifying a role you should pay attention to the sanity & travis CI checks.

**Comment by adi-mohan** [comment]
Going by the Travis CI logs, I think the error caused in one of the builds is "unable to retrieve something" and that is not related to roles that have my changes in them. Mongodb Auth jobs is getting cancelled probably due to timeout issues. 
I think the PR is verified and complete and require your reviews.
Just one thing, if you find this PR suitable to merge, then before/after merging, please add hacktoberfest-accepted label to the PR. Waiting for your review. Thanks!

**Review by cognifloyd — CHANGES_REQUESTED** [review]
Please modify variable access in test_default.py.

#216 was merged, so the mongodb_config role needs these changes as well. Please:
- [add an upstream remote](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork),
- [sync your fork](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork), and then
- make these changes to the mongodb_config role as well.

File: roles/mongodb_linux/molecule/default/tests/test_default.py
Comment (adi-mohan) [reply roles/mongodb_linux/molecule/default/tests/test_default.py]: Thanks for the review. Changes pointed out has been done in latest commit.

File: roles/mongodb_linux/molecule/virtualbox/tests/test_default.py
Comment (adi-mohan) [reply roles/mongodb_linux/molecule/virtualbox/tests/test_default.py]: Thanks for the review. I have synced up the repo and added required changes as well.

File: roles/mongodb_linux/molecule/default/tests/test_default.py
Comment (cognifloyd) [reply roles/mongodb_linux/molecule/default/tests/test_default.py]: Well, bummer. It turns out that testinfra doesn't work like that anymore: https://github.com/philpep/testinfra/issues/447
So, please revert this, as it is causing failures in travis:
https://travis-ci.com/github/ansible-collections/community.mongodb/jobs/402644616#L11410

Thank you for trying it!

Comment (adi-mohan) [reply roles/mongodb_linux/molecule/default/tests/test_default.py:53]: No problem. I have reverted those changes back and Travis also ran fine with the changes. Please let me know if more improvements are needed.

**Comment by adi-mohan** [comment]
Hi! If this PR is eligible to merge, I just wanted a small favour. It would be great if this PR is labeled with hacktoberfest-accepted . Earlier I had asked to mark the issue with the label of hacktoberfest, which you did, but unfortunately the rules have been updated.
I know, I am asking a lot about these labels, here-and-there, but it would help me.
Thanks!

**Review by cognifloyd — APPROVED** [review]
LGTM
Travis tests pass. Merging.

**Pull request merged**
